source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_to_interview_note.jsonnet

```diff
@@ -3,7 +3,7 @@
   id: 'who-to-interview-note',
   content: {
     title: '<em>Interviewer Note</em>',
-    instruction: 'If the respondent was not living at the property on census night, but other current household members were, you must speak to one of those household members instead. <p>If none of those household members are available, you must return to the address to interview one of them at a later date.</p>',
+    instruction: ['If the respondent was not living at the property on census night, but other current household members were, you must speak to one of those household members instead. <p>If none of those household members are available, you must return to the address to interview one of them at a later date.</p>']
```

Comment:
This needs to be split into two separate lines and the `p` tags removed.

Reply:
Think you've commented on an old commit here.